wolox-books/src/App.vue

```diff
@@ -2,14 +2,18 @@
   <div id="app">
     <div id="nav">
       <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
+      <router-link to="/about">About</router-link> |
```

Comment:
why this pipe

Reply:
![image](https://user-images.githubusercontent.com/39703387/50994589-7263ca00-14fb-11e9-88f4-95fab9898081.png)